tasks/rh-sign-image/README.md

```diff
@@ -13,6 +13,9 @@ Task to create internalrequests to sign snapshot components
 | concurrentLimit | The maximum number of images to be processed at once                                      | Yes      | 4                    |
 | pipelineRunUid  | The uid of the current pipelineRun. Used as a label value when creating internal requests | No       |                      |
 
+## Changes in 3.3.0
+* Work in progress for rh-sign-image task.
```

Comment:
?

Reply:
Ah, forgot to remove it. The task was incomplete before the latest commit, so I did not want anyone to review the incomplete task. Fixed.